contrib_bots/lib/github_issues.py

```diff
@@ -0,0 +1,104 @@
+            been typed in correctly, that there is a personal access token
+            with access to public repositories ONLY,
+            and that the repository name is entered correctly.
+            Check ~/.github/github.conf, and make sure there are
+            github_repo   (The name of the repo to post to)
+            github_repo_owner   (The owner of the repo to post to)
+            github_username   (The username of the github bot)
+            github_token    (The personal access token for the github bot)
+            '''
+
+    def triage_message(self, message):
+        # return True if we want to (possibly) respond to this message
+        original_content = message['content']
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups!
+        if message['display_recipient'] == 'issue':
+            return False
+        is_issue = original_content.startswith('@issue')
+        return is_issue
+
+    def handle_message(self, message, client, state_handler):
+
+        original_content = message['content']
+        original_sender = message['sender_email']
+
+        new_content = original_content.replace('@issue', 'by {}:'.format(original_sender,))
+
+        gh = github.Github()
+        # signs into github using the provided username and password
+        gh.auth()
+
+        # Gets rid of the @issue in the issue title
+        issue_title = message['content'].replace('@issue', '').strip()
+        issue_content = ''
+        new_issue_title = ''
+        for part_of_title in issue_title.split():
+            if len(new_issue_title) < self.CHARACTER_LIMIT:
+                new_issue_title = ' '.join([new_issue_title, part_of_title])
+            else:
+                issue_content = ' '.join([issue_content, part_of_title])
+
+        new_issue_title = new_issue_title.strip()
+        issue_content = issue_content.strip()
+        if issue_content:
+            new_issue_title += '...'
+
+        # Creates the issue json, that is transmitted to the github api servers
+        issue = {
+                 'title': new_issue_title,
+                 'body': '{} **Sent by [{}](https://chat.zulip.org/#) from zulip**'.format(issue_content, original_sender),
+                 'assignee': '',
+                 'milestone': 'none',
+                 'labels': [''],
+                }
+        # Sends the HTTP post request
+        r = gh.post(gh.base_url+'/issues', issue)
+
+        if r.ok:
+            # sends the message onto the 'issues' stream so it can be seen by zulip users
+            client.send_message(dict(
+                type='stream',
+                to='issues',
+                subject=message['sender_email'],
+                # Adds a check mark so that the user can verify if it has been sent
+                content='{} :heavy_check_mark:'.format(new_content),
+            ))
+            return
+        else:
```

Comment:
You don't need to include code in `else` block since you are returning from `if` block.